uint8 inputSensor
uint8 operator
float64 comparator_value